raiden/api/rest.py

```diff
@@ -662,11 +681,11 @@ def open(
 
     def connect(
         self,
-        registry_address: typing.TokenNetworkRegistryAddress,
-        token_address: typing.TokenAddress,
-        funds: typing.TokenAmount,
-        initial_channel_target: int = None,
-        joinable_funds_target: float = None,
+        registry_address: TokenNetworkRegistryAddress,
+        token_address: TokenAddress,
+        funds: TokenAmount,
+        initial_channel_target: int = 3,
+        joinable_funds_target: float = 0.4,
```

Comment:
You can use `settings.DEFAULT_JOINABLE_FUNDS_TARGET` and `settings.DEFAULT_INITIAL_CHANNEL_TARGET` for these values

Reply:
Yes, this is unrelated to this PR. Can it be done in a follow up?

Reply:
Sure